### PR TITLE
Create manhuagui

### PR DESCRIPTION
**Pull request by jwstaceyOvO**

### CODE REVIEW

File: data/cygames [hunk 1]
```diff
@@ -3,4 +3,4 @@ cygames.co.jp
 worldflipper.jp
 
 worldflipper.akamaized.net
-umamusume.akamaized.net
+prd-storage-game-umamusume.akamaized.net
```
Comment (IceCodeNew) [inline data/cygames:6]: Please make another PR for this change.

**Review by IceCodeNew — CHANGES_REQUESTED** [review]
.

File: data/manhuagui [hunk 2]
```diff
@@ -0,0 +1,2 @@
+manhuagui.com
+hamreus.com
```
Comment (IceCodeNew) [inline data/manhuagui:2]: image

what the heck is this.

**Review by IceCodeNew — CHANGES_REQUESTED** [review]

**Comment by jwstaceyOvO** [comment]
I really don't know this, I caught 2 domains:i.hamreus.com and
cf.hamreus.com when downloading, so I write them as hamreus.com. If you
mind it, you can just change it as you like.

IceCodeNew ***@***.***> 于2021年12月18日周六 22:14写道：

**Review by IceCodeNew — APPROVED** [review]

**Comment by ayanamist** [comment]
很奇怪 hamreus.com 的问题没人反馈
@IceCodeNew 这个域名看起来是个伪装（你截图的情况），从这个PR至今，一直在用这个域名做图床，而且带有IP校验
image
我是否可以提个PR加进来？

**Pull request merged**